geoengine/workflow.py

```diff
@@ -941,11 +942,14 @@ def __replace_http_with_ws(self, url: str) -> str:
         return f'{ws_prefix}{url_part}'
 
 
-def register_workflow(workflow: Dict[str, Any], timeout: int = 60) -> Workflow:
+def register_workflow(workflow: Union[Dict[str, Any], WorkflowBuilderOperator], timeout: int = 60) -> Workflow:
```

Comment:
Aren't there also examples that could leave the `to_workflow_dict` out?

Reply:
https://github.com/geo-engine/geoengine-python/pull/138/commits/ed5af4d86a4a72b02549a6da630e44379ffa4494